config/software/libxslt.rb

```diff
@@ -57,7 +57,7 @@
   # style path to accomodate
   configure_commands = [
     "--with-libxml-prefix=#{install_dir.sub("C:", "/C")}/embedded",
-    "--with-libxml-include-prefix=#{install_dir}/embedded/include",
+    "--with-libxml-include-prefix=#{install_dir}/embedded/include#{'/libxml2' if version == '1.1.34'}",
```

Comment:
Is this necessary only for libxslt v1.1.34? Or will this be necessary for `libxslt >= 1.1.34`?

Reply:
I have no idea yet.

Reply:
new simpler change seems to work better.